### ISSUE TITLE
Coverage not plotted with plot_fusion()

### ISSUE THREAD

**Issue by csijcs** [stianlagstad/chimeraviz#59]
Love the program. I know this issue was brought up before (almost 1 year ago), but I don't see if there is a solution yet. I can make the fusion plot just fine, but there is no coverage shown. I am using STAR-fusion 1.4.0. to call fusions, and running R 3.5.2 on a Mac OS Mojave 10.14.2. I know you said previously that the tool does not yet support it, but I was hoping that by now it would. Can you please let me know?

Here is my sessioninfo:
`
R version 3.5.2 (2018-12-20)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.2

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
 [1] grid      stats4    parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] Rsamtools_1.34.1       BiocInstaller_1.32.1   chimeraviz_1.8.1       data.table_1.12.0      ensembldb_2.6.6        AnnotationFilter_1.6.0 GenomicFeatures_1.34.3 AnnotationDbi_1.44.0   Biobase_2.42.0         Gviz_1.26.4            GenomicRanges_1.34.0   GenomeInfoDb_1.18.2    Biostrings_2.50.2     
[14] XVector_0.22.0         IRanges_2.16.0         S4Vectors_0.20.1       BiocGenerics_0.28.0   

loaded via a namespace (and not attached):
 [1] ProtGenerics_1.14.0         bitops_1.0-6                matrixStats_0.54.0          bit64_0.9-7                 RColorBrewer_1.1-2          progress_1.2.0              httr_1.4.0                  tools_3.5.2                 backports_1.1.3             DT_0.5                     
[11] R6_2.4.0                    rpart_4.1-13                Hmisc_4.2-0                 DBI_1.0.0                   lazyeval_0.2.1              colorspace_1.4-0            nnet_7.3-12                 tidyselect_0.2.5            gridExtra_2.3               prettyunits_1.0.2          
[21] DESeq2_1.22.2               bit_1.1-14                  curl_3.3                    compiler_3.5.2              htmlTable_1.13.1            DelayedArray_0.8.0          rtracklayer_1.42.1          scales_1.0.0                checkmate_1.9.1             genefilter_1.64.0          
[31] RCircos_1.2.0               stringr_1.4.0               digest_0.6.18               foreign_0.8-71              rmarkdown_1.11              base64enc_0.1-3             dichromat_2.0-0             pkgconfig_2.0.2             htmltools_0.3.6             BSgenome_1.50.0            
[41] htmlwidgets_1.3             rlang_0.3.1                 rstudioapi_0.9.0            RSQLite_2.1.1               gtools_3.8.1                BiocParallel_1.16.6         acepack_1.4.1               dplyr_0.8.0.1               VariantAnnotation_1.28.11   RCurl_1.95-4.11            
[51] magrittr_1.5                GenomeInfoDbData_1.2.0      Formula_1.2-3               Matrix_1.2-15               Rcpp_1.0.0                  munsell_0.5.0               yaml_2.2.0                  stringi_1.3.1               SummarizedExperiment_1.12.0 zlibbioc_1.28.0            
[61] org.Hs.eg.db_3.7.0          plyr_1.8.4                  blob_1.1.1                  crayon_1.3.4                lattice_0.20-38             splines_3.5.2               annotate_1.60.0             hms_0.4.2                   locfit_1.5-9.1              knitr_1.21                 
[71] pillar_1.3.1                geneplotter_1.60.0          biomaRt_2.38.0              XML_3.98-1.17               glue_1.3.0                  evaluate_0.13               biovizBase_1.30.1           latticeExtra_0.6-28         BiocManager_1.30.4          org.Mm.eg.db_3.7.0         
[81] gtable_0.2.0                purrr_0.3.0                 assertthat_0.2.0            ggplot2_3.1.0               xfun_0.5                    xtable_1.8-3                ArgumentCheck_0.10.2        survival_2.43-3             tibble_2.0.1                GenomicAlignments_1.18.1   
[91] memoise_1.1.0               cluster_2.0.7-1             BiocStyle_2.10.0           
`

**Comment by csijcs** [comment]
Sorry, I should also mention that this is the command I am running:
```
plot_fusion(
+  fusion = fusion,
+  bamfile = bamfile,
+  edb = edb)

```
And the output of the parameters:
```
> edb
EnsDb for Ensembl:
|Backend: SQLite
|Db type: EnsDb
|Type of Gene ID: Ensembl Gene ID
|Supporting package: ensembldb
|Db created by: ensembldb package from Bioconductor
|script_version: 0.0.1
|Creation time: Fri Feb 22 11:30:47 2019
|ensembl_version: 95
|ensembl_host: unknown
|Organism: Homo_sapiens
|genome_build: GRCh38
|DBSCHEMAVERSION: 1.0
|source_file: Homo_sapiens.GRCh38.95.gtf
| No. of genes: 58735.
| No. of transcripts: 206601.
```

```
>bamfile
[1] "/Users/csijcs/RHB5885_combined_junction_and_spanning_reads.bam"
```

```
> fusion
[1] "Fusion object"
[1] "id: 525"
[1] "Fusion tool: starfusion"
[1] "Genome version: hg38"
[1] "Gene names: JARID2-HNRNPA2B1"
[1] "Chromosomes: chr6-chr7"
[1] "Strands: +,-"
[1] "In-frame?: NA"
```

**Comment by stianlagstad** [comment]
Hey @csijcs! Are you referring to this issue? https://github.com/stianlagstad/chimeraviz/issues/10

**Comment by csijcs** [comment]
Yes. Sorry, wasn't sure if it was better start a new issue or comment on an old one.  Still seems to be an issue i think?

**Comment by stianlagstad** [comment]
Thanks for the quick response! Yes, the issue is still there. Knowing that a user is waiting for a fix is great motivation to look closer at it, so I'll try to do that. I don't have an estimate for when that will be though.

**Comment by csijcs** [comment]
Ok, thanks for the info. I'll be looking forward to it.

**Comment by stianlagstad** [comment]
Hi @csijcs,

It turns out that https://github.com/stianlagstad/chimeraviz/issues/10 was just due to me using the wrong file in a couple of method-calls in the vignette. So I think that the issue you're seeing is something else. Would you be able to share the files you're using? Then I can look into it.

**Comment by stianlagstad** [comment]
I believe the fix for #75 solves this one as well. Please comment again if that's not the case.
